closed/Intel/code/dlrm-99.9/pytorch-cpu/README.md

```diff
@@ -127,3 +127,22 @@ please follow the #step7- Run command for server and offline mode above
 ```
 
 
+# Run on AWS
+```
+./run_docker.sh 038cf950d509  /workspace/RUN.sh  /opt/workdir/code/dlrm-99.9/pytorch-cpu aws_dlrm
```

Comment:
we won't release AWS image so users couldn't access this docker image id.
suggest to remove AWS session and keep it within TCE team.

Reply:
Since you suggest to remove AWS session, do you still want me to merge the code changes to "aws" branch?